packages/elements/src/components/ino-table-header-cell/ino-table-header-cell.tsx

```diff
@@ -216,7 +216,6 @@ export class InoTableHeaderCell implements ComponentInterface {
           <ino-popover
             trigger="focus click"
             placement="bottom-start"
-            color-scheme="transparent"
```

Comment:
Why you removed this? Only `'secondary' & 'light'` was removed from the `ino-popover`